_maps/map_files/Atlas/job_changes.dm

```diff
@@ -70,4 +70,28 @@ MAP_REMOVE_JOB(brig_phys)
     total_positions = 1
     spawn_positions = 1
 
+/datum/job/virologist/New()
+    ..()
+    MAP_JOB_CHECK
+    total_positions = 1
+    spawn_positions = 1
+
+/datum/job/paramedic/New()
+    ..()
+    MAP_JOB_CHECK
+    total_positions = 2
+    spawn_positions = 2
+	
+/datum/job/medical_doctor/New()
+    ..()
+    MAP_JOB_CHECK
+    total_positions = 3
```

Comment:
3 to troche malo, i tak z reguly nie ma tyle osób, a jak już są to niech grają czym chcą w sumie